pkg/util/credential.go

```diff
@@ -48,6 +48,11 @@ var (
 			Verbs:           []string{"get"},
 		},
 	}
+
+	// managedByKarmadaLabels: set the label to indicate that the resource is created by Karmada.
+	managedByKarmadaLabels = map[string]string{
```

Comment:
The variable `managedByKarmadaLabels` is extracted here. Would it be better if it was exported so that other files can also use it? Like, https://github.com/karmada-io/karmada/blob/f03b5e9803e4738706bb6fcf807350a55a4ad703/pkg/controllers/certificate/cert_rotation_controller.go#L239-L245 could be changed to 
```go
	certificateSigningRequest := &certificatesv1.CertificateSigningRequest{
		ObjectMeta: metav1.ObjectMeta{
			Name: csrName,
			Labels: util.ManagedByKarmadaLabels,
			},
		},
```

Reply:
Good suggestion, implemented ✅